src/example/rendering_particle.cpp

```diff
@@ -12,28 +13,34 @@
 #include <crepe/api/Sprite.h>
 #include <crepe/api/Texture.h>
 #include <crepe/api/Transform.h>
-#include <crepe/api/Vector2.h>
 #include <crepe/system/RenderSystem.h>
+#include <crepe/types.h>
 
 #include <chrono>
-#include <iostream>
 #include <memory>
 
 using namespace crepe;
 using namespace std;
 
 int main(int argc, char * argv[]) {
 	ComponentManager mgr;
-	GameObject game_object = mgr.new_object("", "", vec2{0, 0}, 0, 0.1);
+	GameObject game_object = mgr.new_object("", "", vec2{0, 0}, 0, 1);
 	RenderSystem sys{mgr};
 	ParticleSystem psys{mgr};
+	AnimatorSystem asys{mgr};
 
 	Color color(255, 255, 255, 255);
```

Comment:
Waarom lijkt het geen effect te hebben als ik Color aanpas?
![image](https://github.com/user-attachments/assets/2c774e0e-cc7d-4e2d-b486-a699b8dfd7f0)

Reply:
dit is nog in een andere PR